README.md

```diff
@@ -6,6 +6,23 @@ with tket, a quantum computing toolkit and optimising compiler developed by Quan
 [QuEST](https://quest.qtechtheory.org/) is an open-source high performance simulator of
 quantum circuits, state-vectors and density matrices.
 
```

Comment:
Could we use a structure for the README.md similar to this one: https://github.com/CQCL/pytket-qulacs/blob/develop/README.md ? We can do that in another PR though.